requirements/requirements_docs.txt

```
@@ -1,5 +1,5 @@
 docutils>=0.17
-sphinx~=5.1.1
+sphinx~=5.2.3
```

```suggestion
sphinx==5.2.3
```
we may want to pin this version, since they seem to be doing more than just bugfixes in patch versions recently